packages/frontend/src/pages/Search/SearchResult.tsx

```diff
@@ -1,24 +1,35 @@
 import { Alert, Typography } from '@mui/material';
-import { useSearchParams } from 'react-router-dom';
 import CircularProgress from '@mui/material/CircularProgress';
 import { useTranslation } from 'react-i18next';
 
 import { Tags } from '../../components/Tags';
 import { ContainerWrapper } from '../Landing/index.styles';
 import { NodeItem } from './NodeItem';
 import { usePostAlfrescoSearch } from '../../hooks/query/Search';
+import { useContext } from 'react';
+import { SearchContext } from '../../contexts/SearchContext';
+import { formatYear } from '../../utils/helpers';
+import { useSearchParams } from 'react-router-dom';
 
 export const SearchResult = () => {
   const { t } = useTranslation(['search', 'common']);
   const [searchParams] = useSearchParams();
   const query = searchParams.get('query');
+  const searchContext = useContext(SearchContext);
+  const { years, checkedList } = searchContext;
+  const searchParameter = {
```

Comment:
Could this use the type `TAlfrescoSearchProps` that's currently in `packages/frontend/src/hooks/query/Search.ts`?